src/components/ChatContent/ToolsContent.tsx

```diff
@@ -118,7 +118,13 @@ export const ToolContent: React.FC<{
 
   return (
     <Container>
-      <Collapsible.Root open={open} onOpenChange={setOpen}>
+      <Collapsible.Root
```

Comment:
Can thie be wrapped in `ScrollArea` that way it's consistent with the other scroll bars.